app/models/badges/v2_badge.rb

```diff
@@ -0,0 +1,21 @@
+module Badges
+  class V2Badge < Badge
+    seed "V2",
+      :common,
+      'v2',
+      'Awarded for having joined Exercism v2'
+
+    def award_to?(user)
+      return false if user.confirmed_at.nil?
+
+      user.created_at < V3_RELEASE_DATE
+    end
+
+    def send_email_on_acquisition?
+      false
+    end
+
+    V3_RELEASE_DATE = Time.new(2021, 9, 1, 0, 0, 0, 0).freeze
```

Comment:
I've just set the time to the first second, as we had frozen the website before launch anyway so anyone that registered that day must be a v3 user.